packages/integration-tests/src/tests/ui-cloud/smoke.test.ts

```diff
@@ -68,6 +68,79 @@ describe('smoke testing for cloud', () => {
     expect(page.url()).toBe(new URL(`/${tenantId ?? ''}/onboarding/welcome`, logtoCloudUrl).href);
   });
 
+  it('can complete the onboarding welcome process and enter the user survey page', async () => {
+    // Select the project type option
+    const personalOption = await page.waitForSelector(
+      'div[role=radio]:has(input[name=project][value=personal])'
+    );
+    await personalOption.click();
```

Comment:
i don't think you need to wait and click since the other cases are already using `.toClick()` and so for convenience. and even at the top of this file we have a comment for this

<img width="754" alt="image" src="https://user-images.githubusercontent.com/14722250/223993529-7d854763-c4dd-4b13-8ce6-bbf7a10f0425.png">